src/components/auth/AuthForm.js

```diff
@@ -8,15 +8,14 @@ export const AuthForm = ({
   return (
     <>
       <form onSubmit={onSubmit}>
-        <div>
-          <label htmlFor="email">Email</label>
-          <input type="email" name="email" id="email" />
-        </div>
+        <InputAuthText value="Email" name="email" type="email" id="email" />
```

Comment:
skoro name, type i id to ta sama wartosc to mozna przekazac jednego propsa zamiast trzech

Reply:
racja, też się nad tym zastanawiałam, właściwie zawsze te wartości są takie same więc wrzuciłam je już do jednego propsa